custom_components/adaptive_lighting/switch.py

```diff
@@ -1648,6 +1658,10 @@ def __init__(self, hass: HomeAssistant):
         # Track light transitions
         self.transition_timers: dict[str, _AsyncSingleShotTimer] = {}
 
+        # When a state is different `max_cnt_significant_changes` times in a row,
+        # mark it as manually_controlled.
+        self.max_cnt_significant_changes = 2
```

Comment:
Are you adding this back? Or accidental?

Reply:
Accidental, happened during the merge!